### PR TITLE
[memgraph-2-15<] System replication

### PR DESCRIPTION
**Pull request by andrejtonev**
### Description

Please briefly explain the changes you made.

### Pull request type

Please check what kind of PR this is:

- [ ] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to (these are all the related PRs in ordered from newer; older PRs mostly made changes that were invisible to the users): 
[System replication experimental flag #1702](https://github.com/memgraph/memgraph/pull/1702)
[Adding authentication data replication #1666](https://github.com/memgraph/memgraph/pull/1666)
[Fix REPLICA timestamps #1615](https://github.com/memgraph/memgraph/pull/1615)
[Replication refactor part 7 #1550](https://github.com/memgraph/memgraph/pull/1550)
[Bugfix: correct replication handler #1540](https://github.com/memgraph/memgraph/pull/1540)
[Mg replication refactor part 6 again #1484](https://github.com/memgraph/memgraph/pull/1484)
[Replication refactor (part 5) #1378](https://github.com/memgraph/memgraph/pull/1378)
[Fix subtle replication bug #1370](https://github.com/memgraph/memgraph/pull/1370)
[Decouple pure replication state from storage #1325](https://github.com/memgraph/memgraph/pull/1325)
[Refactor replication client/server for easier construction #1311](https://github.com/memgraph/memgraph/pull/1311)
[Refactor interpreter to support multiple distributed clocks (Part 1) #1281](https://github.com/memgraph/memgraph/pull/1281)
[Replication refactor (part 4) #1211](https://github.com/memgraph/memgraph/pull/1211)
[Decoupling Interpreter from Storage #1186](https://github.com/memgraph/memgraph/pull/1186)
[Replication refactor (part 3) #1177](https://github.com/memgraph/memgraph/pull/1177)
[Decoupling replication logic from InMemoryStorage #1169](https://github.com/memgraph/memgraph/pull/1169)
[Refactor memgraph.cpp #1164](https://github.com/memgraph/memgraph/pull/1164)

### Checklist:

- [ ] Check all content with Grammarly
- [ ] Perform a self-review of my code
- [ ] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors
- [ ] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

File: pages/configuration/replication.mdx [hunk 1]
```diff
@@ -15,10 +15,10 @@ Memgraph 2.9 introduced a new configuration flag
 If you want instances to remember their role and configuration in a replication
 cluster upon restart, the `--replication-restore-state-on-startup` needs to be
 set to `true` when first initializing the instances and remain `true` throughout
-the instances' lifetime. 
+the instances' lifetime.
```
Comment (antoniofilipovic) [inline pages/configuration/replication.mdx:18]: @andrejtonev this resolves this PR: https://github.com/memgraph/memgraph/pull/1707, right?

**Comment by antoniofilipovic** [comment]
Add this PR to the list: https://github.com/memgraph/memgraph/pull/1707

File: pages/configuration/replication.mdx (hunk 1 above)
Comment (andrejtonev) [reply pages/configuration/replication.mdx:18]: I think we need to explicitly tell the user they need to re-register on MAIN a replica if running without this flag.
Right?

File: pages/help-center/faq.md [hunk 2]
```diff
@@ -272,8 +272,14 @@ You can use Memgraph Lab, a visual user interface that enables you to:
 
 ### Does replication affect performance?
 
-Replication should not in any way affect the performance of your database
-instance.
+Replication can impact performance in several ways. 
+
+When using SYNC replica, the replication is done at time of commit. This means
+that each commit takes longer, as the data needs to be replication before the
```
Comment (antepusic) [inline pages/help-center/faq.md:278]:
```suggestion
that each commit takes longer, as the data needs to be replicated before the

```

File: pages/configuration/replication.mdx (hunk 1 above)
Comment (kgolubic) [reply pages/configuration/replication.mdx:18]: > I think we need to explicitly tell the user they need to re-register on MAIN a replica if running without this flag. Right?

@antoniofilipovic and  @andrejtonev are any additional documentation updates need when it comes to this or can I merge the docs?

Comment (antoniofilipovic) [reply pages/configuration/replication.mdx:18]: Feel free to merge, I will open new PR for this

**Review by kgolubic — APPROVED** [review]
Everything looks OK.

File: pages/configuration/replication.mdx (hunk 1 above)
Comment (antoniofilipovic) [reply pages/configuration/replication.mdx:18]: Done in new PR

**Pull request merged**